cmd/dump.go

```diff
@@ -28,7 +28,8 @@ It can be used for backup and capture Gogs server image to send to maintainer`,
 	Action: runDump,
 	Flags: []cli.Flag{
 		stringFlag("config, c", "custom/conf/app.ini", "Custom configuration file path"),
-		boolFlag("verbose, v", "show process details"),
+		boolFlag("verbose, v", "Show process details"),
+		stringFlag("output, o", os.TempDir(), "Destination dir path"),
```

Comment:
`tempdir, t` makes more sense since that's what you're actually changing
(not sure if this needs a short-hand version though...) @Unknwon